### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by kennethlimcp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ the spec is finalized.  However, we encourage you to try out the tool and give f
 ```bash
 # create a 'github.com/opencontainers' in your GOPATH
 cd github.com/opencontainers
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc`b
```
Comment (mrunalp) [inline README.md]: There is an additional `b at the end of the line.

**Comment by clycheng2015** [comment]
First time come here

File: README.md (hunk 1 above)
Comment (kennethlimcp) [inline README.md]: @mrunalp. Sorry for the mess. It's now done!

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**